features/built_in_matchers/all.feature

```diff
@@ -17,6 +17,9 @@ Feature: `all` matcher
     expect([1, 4, 21]).to all( be_odd.or be < 10 )
     ```
 
+  If you are looking for "any" member of a collection that passes an
+  expectation, look at the `include`-matcher.
```

Comment:
Relish wraps at the same places we do in the source, unfortunately, so we're moving to making each paragraph one lone unwrapped line in the source, which allows normal HTML flow to wrap it so that it looks good.